packages/react-native/src/primitives/Checkbox/Checkbox.tsx

```diff
@@ -67,22 +61,20 @@ export default function Checkbox<T>({
     [buttonStyle]
   );
 
+  // TODO: replace View with Pressable and IconButton with icon once Icon primitive is added
```

Comment:
Can we replace the `View` now and just pass `disabled={true}` to the `IconButton`?

Reply:
I'm not really seeing the benefit of that, will need to come back and refactor this anyway.

Reply:
IMO the benefit is that it is completed in this PR rather than a follow up, and the follow up is just swapping the `IconButton` out for the `Icon`. I don't feel strongly or anything though

Reply:
sure. done!